platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -182,10 +182,10 @@ private Browser(final Set<Integer> localNodesToStart) throws IOException {
                 Settings.getInstance().getConfigPath());
 
         final ConfigSource settingsConfigSource = LegacyFileConfigSource.ofSettingsFile();
-        final ConfigSource settingsAliasConfigSource = ConfigAliases.addConfigAliases(settingsConfigSource);
+        final ConfigSource mappedSettingsConfigSource = ConfigMappings.addConfigMapping(settingsConfigSource);
 
         final ConfigSource configPropertiesConfigSource = new ConfigPropertiesSource(configurationProperties);
-        final ConfigSource configPropertiesAliasConfigSource = new AliasConfigSource(configPropertiesConfigSource);
+        final ConfigSource mappedConfigPropertiesConfigSource = new MappedConfigSource(configPropertiesConfigSource);
```

Comment:
Why do we map the source for the config.txt when we do not add a mapping to it?

Reply:
Makes no sense, is out